src/app/pages/user/dialogs/update/update-user.dialog.ts

```diff
@@ -1,17 +1,34 @@
 import { Component, Inject } from '@angular/core';
 import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material';
-import { FormControl, Validators } from '@angular/forms';
+import { UserType } from '../../../../models/user.model';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { ClassService } from '../../../class/services/class.graphql.service';
+import { Class } from '../../../../models/class.model';
 
 @Component({
   selector: 'app-update-user.dialog',
   templateUrl: './update-user.dialog.html',
   styleUrls: ['./update-user.dialog.scss'],
 })
 export class UpdateUserDialogComponent {
-  constructor(public dialogRef: MatDialogRef<UpdateUserDialogComponent>, @Inject(MAT_DIALOG_DATA) public data: any) {}
+  form: FormGroup;
+  roles: string[];
+  userRoleEnum = UserType;
```

Comment:
just for the sake of style, can you please initialize `userRoleEnum` inside the constructor with the rest of the variables? thanks